modules/client/test/unit/context-plugins.test.ts

```diff
@@ -23,33 +30,51 @@ describe("ContextPlugin instances", () => {
     contextParams = {
       network: "mainnet",
       signer: new Wallet(TEST_WALLET),
-      daoFactoryAddress: "0x1234",
+      daoFactoryAddress: "0x1234567890123456789012345678901234567890",
       web3Providers: web3endpoints.working,
       gasFeeEstimationFactor: 0.1,
-      graphqlNodes: [],
+      graphqlNodes: [{ url: "https://example.com" }],
+      ipfsNodes: [{ url: "https://example.com" }],
     };
   });
-  it("Should create an empty context", () => {
-    const context = new ContextPlugin({});
-
-    expect(context).toBeInstanceOf(ContextPlugin);
-  });
   it("Should create an empty context and have default values", () => {
-    const context = new ContextPlugin({});
+    const context = new ContextPlugin();
```

Comment:
Is it still possible to create a context without a list of web3 endpoints? I thought there were some required params, at least

Reply:
Yes, I didnt make it required because that was the aproach that we used with the signer, if we want the context to crash when empty, I'd do it equally for both cases.